csvcubed/tests/unit/cube/qb/test_qb_unit.py

```diff
@@ -6,18 +6,18 @@
 
 def test_new_unit_base_unit_validation():
     """
-    Ensure that if either of :obj:`base_unit` or :obj:`base_unit_scaling_factor` are specified, and the other isn't,
+    Ensure that if :obj:`base_unit_scaling_factor` is specified and :obj:`base_unit` isn't,
     the user gets a suitable validation error.
     """
-    _assert_both_properties_defined_error(
-        NewQbUnit(
-            "Some Unit",
-            base_unit=ExistingQbUnit("http://existing-unit"),
-            base_unit_scaling_factor=None,
-        ),
-        "base_unit",
-        "base_unit_scaling_factor",
-    )
+    # _assert_both_properties_defined_error(
```

Comment:
Should have an issue in comment saying why it's disabled and when to re-enable it.